build.gradle

```diff
@@ -285,8 +285,6 @@ acceptanceTest {
     testLogging {
         events "passed", "skipped", "failed"
     }
-    testLogging.showStandardStreams = true
-    testLogging.exceptionFormat = 'full'
```

Comment:
@v-stepanov maybe we can at least leave 'testLogging.exceptionFormat = 'full'? I would like to see the reason of the failing test when I run tests from console.